### DISCUSSION TITLE
2 way audio? Homekit

### DISCUSSION THREAD

**Discussion by marksowell890** [briis/unifiprotect#231] [Q&A]
Im figuring things out.  this work on unifi protect is awesome.  very impressive.      I have things working in hASS with a G4 doorbell.   Sound is working in the HASS.  Not sure how to test 2 way yet.    Does anyone have pointers on setting Is might need to set up for 2 way audio to work with he homkit integration as well?

**Comment by bdraco** [comment]
https://github.com/hjdhjd/homebridge-unifi-protect

This project has implemented it.  

I don't know of any project that integrations 2-way audio with Home Assistant

**Reply by marksowell890** [reply]
thank you
